hub/policy/src/main/java/uk/gov/ida/hub/policy/exception/PolicyApplicationExceptionMapper.java

```diff
@@ -61,18 +58,7 @@ private boolean auditException(final ApplicationException exception) {
         boolean isAudited = exception.isAudited();
 
         if (!isAudited && exception.requiresAuditing()) {
-            EventDetails eventDetails = new EventDetails(
-                    downstream_uri,
-                    exception.getUri().or(URI.create("uri-not-present")).toASCIIString());
-
-            if (getSessionId().isPresent()) {
-                eventSinkMessageSenderFacade.audit(exception, exception.getErrorId(), getSessionId().get(), eventDetails);
-            } else {
-                eventSinkMessageSenderFacade.audit(exception, exception.getErrorId(), NO_SESSION_CONTEXT_IN_ERROR,
-                        eventDetails
-                );
-            }
-
+            eventLogger.logErrorEvent(exception.getErrorId(), getSessionId().or(NO_SESSION_CONTEXT_IN_ERROR), exception.getMessage(), exception.getUri().or(URI.create("uri-not-present")).toASCIIString());
```

Comment:
❤️ simplification

Reply:
but I would break this line in multilines, it's too long and hard to see where the "or"s relate to

Reply:
I have fixed it in a new pull request (https://github.com/alphagov/verify-hub/pull/30).